doc/changelog.md

```diff
@@ -137,75 +227,81 @@
       return qml.probs()
   ```
 
-  would no longer decompose the `PSWAP` and `ISWAP` gates to `SWAP`s, `CNOT`s and `Hadamard`s. Instead it would just call Braket's native `PSWAP` and `ISWAP` gates at runtime.
-  [(#458)](https://github.com/PennyLaneAI/catalyst/pull/458)
+  would no longer decompose the `PSWAP` and `ISWAP` gates.
 
-* Add support for the `BlockEncode` operator in Catalyst.
+* The `qml.BlockEncode` operator is now supported with Catalyst.
   [(#483)](https://github.com/PennyLaneAI/catalyst/pull/483)
 
-* Remove copies of TOML device configuration files for Lightning device in Catalyst.
-  [(#472)](https://github.com/PennyLaneAI/catalyst/pull/472)
+* Additional debugging functions are now available in the `catalyst.debug` directory.
+  [(#529)](https://github.com/PennyLaneAI/catalyst/pull/529)
+  [(#522)](https://github.com/PennyLaneAI/catalyst/pull/531)
 
-* Remove `qextract_p` and `qinst_p` from forced-order primitives.
-  [(#469)](https://github.com/PennyLaneAI/catalyst/pull/469)
+  This includes:
 
-* Update `AllocateQubit` and `AllocateQubits` in `LightningKokkosSimulator` to preserve
-  the current state-vector before qubit re-allocations.
-  [(#479)](https://github.com/PennyLaneAI/catalyst/pull/479)
+  - `compile_from_mlir(ir[, compiler, ...])` to compile a Catalyst function to binary code from the
+    provided MLIR.
 
-* Add support for post-selection in mid-circuit measurements. This is currently supported only on Lightning simulators.
-  [(#491)](https://github.com/PennyLaneAI/catalyst/pull/491)
+  - `filter_static_args(args, static_argnums)` to remove static values from arguments using the
+    provided index list.
 
-  This is an example of post-selection usage:
+  - `get_cmain(fn, *args)` to return a C program that calls a jitted function with the provided
+    arguments.
 
-  ```py
-  import pennylane as qml
-  from catalyst import qjit
+  - `print_compilation_stage(fn, stage)` to print one of the recorded compilation stages for a
+    JIT-compiled function.
 
-  dev = qml.device("lightning.qubit", wires=1)
+  - `promote_arguments(target_signature, args)` to promote arguments to the provided target
+    signature, preserving PyTrees.
 
-  @qjit
-  @qml.qnode(dev)
-  def f():
-      qml.Hadamard(0)
-      m = measure(0, postselect=1)
-      return qml.expval(qml.PauliZ(0))
-  ```
+  For more details, please see the `catalyst.debug` documentation.
 
-* Add support for reset option in mid-circuit measurements.
-  [(#507)](https://github.com/PennyLaneAI/catalyst/pull/507)
+* Remove copies of TOML device configuration files for Lightning device in Catalyst.
+  [(#472)](https://github.com/PennyLaneAI/catalyst/pull/472)
```

Comment:
@erick-xanadu @maliasadi what does this entry mean?

Reply:
There was a transition period where we needed copies of these files in the Catalyst repo. Now that the files live in PL-lightning, there's no reason to have them here.

Reply:
```suggestion
* Remove redundant copies of TOML files for `lightning.kokkos` and `lightning.qubit`.
  [(#472)](https://github.com/PennyLaneAI/catalyst/pull/472)
  
  `lightning.kokkos` and `lightning.qubit` now ship with their own TOML file. As such, we use the TOML file provided by them.
```